### ISSUE TITLE
Missing Trash Can in Menu Bar

### ISSUE THREAD

**Issue by LinuxUser3** [linuxmint/xviewer#46]
```
 * Xviewer version (xviewer --version) - 1.4.3
 * Distribution - Mint 18.2
```

**Issue**

Is it possible to adjust the menu bar in Xviewer 1.4.3? - The viewer is fine, but I miss the trash can in the menu bar. Yes, one can click on "Select --> Delete Picture", but this is time consuming while reviewing pictures.

**Steps to reproduce**
./.

**Expected behaviour**
./.

**Other information**
./.

**Comment by bjd-pfq** [comment]
I second that request. It used to be there, but went missing. It's an essential feature, I would think.
Please respond.

**Comment by bjd-pfq** [comment]
The trash can icon and functionality is still missing.

**Comment by firewiremb** [comment]
More than five years have passed since then. Is there still hope that it will be implemented? It's a fundamental feature of an image viewer, I think. In any case, images cannot be sorted within a reasonable period of time with this image viewer.

**Comment by thegame1999** [comment]
This can be closed. Trash action can be accessed with the Edit menu or Del key.

**Comment by firewiremb** [comment]
I still disagree. While you can delete files with the Delete key, for someone like me who prefers using the mouse, it would be very helpful to have a trash can button right next to the forward and back arrows.

**Comment by bjd-pfq** [comment]
I agree with firewiremb. I am a tetraplegic, my fingers are permanently curled inwards, and I do all of my movement with the mouse. To have to reach for the keyboard and try to hit a key with my curled finger is a royal pain in the backside, and a design hostile to me.
User-'frienliness' that only addresses the norm is in fact user-aggression.

**Comment by thegame1999** [comment]
I'm sorry to hear. Right clicking anywhere on the image and hitting "Move to Trash" on the context menu is another option to the Edit menu.

**Comment by bjd-pfq** [comment]
When you translate your sympathy in implementing the requested functionality, it saves me from 50% of the clicks.

**Comment by DamianMalik** [comment]
@thegame1999 :  So it took you nine years to come up with that answer? Sad to hear. 
I will look for another image viewer (that will also allow me to turn off dark mode).
